src/models/mint.ts

```diff
@@ -1,4 +1,14 @@
-export interface MintData {
+import { BigNumberish } from "ethers";
+import { ItemTransfer } from "./transfers";
+
+export type TokenType = "ERC721" | "ERC1155";
+
+export interface PartOwner {
```

Comment:
I renamed this interface as Rarible describe it as `Part`, which without any context is hard to guess, I think `PartOwner` is a much appropriate name and could be easily understood by Rarible's community.
What are your thoughts?